rfcs/SchemaCoordinates.md

```diff
@@ -0,0 +1,372 @@
+# RFC: Schema Coordinates
+
+**Proposed by:** [Mark Larah](https://twitter.com/mark_larah) - Yelp
+
+This RFC proposes formalizing "Schema Coordinates" - a human readable syntax to
+uniquely identify a type, field, or field argument defined in a GraphQL Schema.
+
+This should be listed as a non-normative note in the GraphQL specification to
+serve as an official reference for use by third party tooling.
+
+## 📜 Problem Statement
+
+Third party GraphQL tooling and libraries may wish to refer to a field, or set of
+fields in a schema. Use cases include documentation, metrics and logging
+libraries.
```

Comment:
GraphiQL isn't third party, and still wants to refer to this, so I've removed "Third party". Also made the intention broader.

```suggestion
GraphQL tooling and libraries may wish to refer to the various components of a GraphQL schema. Use cases include documentation, metrics and logging
libraries.
```